imap_processing/cdfutils/global_base.py

```diff
@@ -0,0 +1,353 @@
+from dataclasses import dataclass
+
+import numpy as np
+
+from imap_processing.cdfutils.defaults import GlobalConstants
+
+
+class Epoch:
```

Comment:
For these global things with static methods, what if we just made them a dictionary? So assigned them to a global constant `EPOCH = {...}` and then later you wouldn't have to reference the `.output()` call from the method, but you could just pipe this global constant in directly.

Reply:
Sure, I put them as classes in case people want to overwrite them, but I'm not sure if that's actually something that will happen (especially for some of the global attributes.) I can move some of them if they aren't going to change though